Sender.cpp

```diff
@@ -0,0 +1,57 @@
+#include"stream_bms.h"
+
+void decrement_to_zero(unsigned counter)
```

Comment:
Name functions according to their purpose, not implementation.
Purpose here is to delay / sleep.